reconcile-tags/reconcile-tags.py

```diff
@@ -0,0 +1,82 @@
+"""Reads json files mapping docker digests to tags and reconciles them.
+
+Reads all json files in current directory and parses it into repositories
+and tags. Calls gcloud beta container images add-tag on each entry.
+If there are no changes that api call is no-op.
+"""
+
+import argparse
+import glob
+import json
+import logging
+import subprocess
+
+
+def call(command, dry_run):
+    if not dry_run:
+        logging.debug('Running {0}'.format(command))
+        output = subprocess.check_output([command], shell=True)
+        logging.debug(output)
+        return output
+    else:
+        logging.debug('Would have run {0}'.format(command))
+
+
+def add_tags(digest, tag, dry_run):
+    logging.debug('Tagging {0} with {1}'.format(digest, tag))
+    command = ('gcloud beta container images add-tag {0} {1} '
+               '-q'.format(digest, tag))
+    call(command, dry_run)
+
+
+def delete_tag(repo, tag, dry_run):
+    full_tag = repo + ":" + tag
+    logging.debug('Removing {0}'.format(full_tag))
+    command = ('gcloud beta container images delete {0} -q'.format(full_tag))
+    call(command, dry_run)
+
+
+def reconcile_tags(dry_run):
+    call('gcloud config list', False)
```

Comment:
Nice, a comment explaining why we hardcode dry-run to False for this line would be nice.